tests/integration/test_auth.py

```diff
@@ -17,14 +17,42 @@ def test_user_registration(client):
     url = reverse('auth-register')
     credentials = {
         'email': 'test@test.com',
-        'password': 'localhost'
+        'password': 'localhost',
```

Comment:
Let's not use any parameters here at first, to ensure that only with `email` and `password` I can sign-up.

Then write a test, which can test all other attributes are set-up if they are provided.